pkg/volume/csi/csi_attacher.go

```diff
@@ -197,8 +197,18 @@ func (c *csiAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.No
 		}
 
 		attachID := getAttachmentName(volumeHandle, driverName, string(nodeName))
+		var attach *storage.VolumeAttachment
+		if c.plugin.volumeAttachmentLister != nil {
```

Comment:
Is the only time this is nil in unit tests?

Reply:
adding different test-only code paths doesn't seem like a good idea, especially if it makes unit tests behave significantly differently than real runs